src/services/models/Schema.ts

```diff
@@ -152,6 +154,10 @@ export class SchemaModel {
       return;
     }
 
+    if (schema.if || schema.then || schema.else) {
```

Comment:
I think we should union this with `if ((!ifOperator && !thenOperator) || (!ifOperator && !elseOperator)) return;` from `initConditionalOperators`
I mean better to have 1 condition in 1 place

Reply:
@zalesky Thank for good suggestion. Improved.